language/tests/gapic/v1/language_service_smoke_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
For some reason, the person who wrote the system tests in the auto-gen called them "smoke tests". I have a request to rename this.

Right now, I am fairly certain it will run never. I will fix this at a future point.